### ISSUE TITLE
option npm doesnt seem to work for me

### ISSUE THREAD

**Issue by Grievous-Spinout** [strapi-community/strapi-tool-dockerize#65]
## 🤷‍♀️ What did you do
ran: npx @strapi-community/dockerize new
--projectname="xyxyxyxy"
--dbtype="postgres"
--dbhost="localhost"
--dbport=5432
--dbname="xyxyxyxy"
--dbusername="yxyxyxyxyx"
--dbpassword="yxyxyxyxyx"
--projecttype="js"
--packagemanager="npm"
--usecompose=true
--env="development"

![image](https://user-images.githubusercontent.com/10957952/226181550-7827f863-a13d-4a83-b6ec-828270001f82.png)
it seems to detect yarn instead of npm.

## 🙇‍♀️ Expected behavior/code
Expected it to detect npm

## 👩‍💻 Environment
linux pop_os

**Comment by Eventyret** [comment]
Thank you it defaults to yarn though I agree it should be npm.

It's not detecting anything as it hasn't detected a strapi application.
Please do try create a strapi application first then run the tool in the strapi application folder. And let me know.

**Comment by SudeepPatel-0812** [comment]
Hi @Grievous-Spinout Thanks for adding your issue. 

I tried to reproduce your issue and for me what happened was there was a yarn.lock in my directory then it detects yarn but if there is no yarn.lock and yarn is not installed on your machine it detects npm.

I have attached a screenshot for your reference.

![Screenshot 2023-03-24 at 11 55 57 PM](https://user-images.githubusercontent.com/93346617/227610491-61ff839a-9541-43b5-af99-76eb28558ae2.jpeg)

I hope that helps :)
